front/app/modules/commercial/customizable_navbar/admin/containers/NewPageForm/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// services
+import { createPage } from '../../../services/pages';
+import { handleAddPageFiles } from 'services/pageFiles';
+
+// hooks
+import useAppConfigurationLocales from 'hooks/useAppConfigurationLocales';
+import usePageSlugs from 'hooks/usePageSlugs';
+
+// components
+import { Formik, FormikProps } from 'formik';
+import GoBackButton from 'components/UI/GoBackButton';
+import PageForm, { FormValues, validatePageForm } from 'components/PageForm';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../messages';
+
+// utils
+import clHistory from 'utils/cl-router/history';
+import { isNilOrError } from 'utils/helperUtils';
+import getInitialValues from './getInitialValues';
+import { NAVIGATION_PATH } from '..';
+
+const PageTitle = styled.h1`
+  width: 100%;
+  font-size: 2rem;
+  margin: 1rem 0 3rem 0;
+`;
+
+const NewPageForm = () => {
+  const appConfigurationLocales = useAppConfigurationLocales();
+  const pageSlugs = usePageSlugs();
+
+  if (isNilOrError(appConfigurationLocales) || isNilOrError(pageSlugs)) {
+    return null;
+  }
+
+  const goBack = () => {
+    clHistory.push(NAVIGATION_PATH);
+  };
+
+  const handleSubmit = async (
+    values: FormValues,
+    { setSubmitting, setStatus }
+  ) => {
+    const localPageFiles = values.local_page_files;
+
+    try {
+      const page = await createPage(values);
```

Comment:
I guess that's ok sine it works, but would be best practise to not include the page files in the page call since thay're not accepted. If we modify the page endpoint to accept the files, this might cause trouble and send them twice or something. (Unlikely though, so this is very low priority)